examples/wheel/wheel_test.py

```diff
@@ -450,15 +412,15 @@ def test_rule_creates_directory_and_is_included_in_wheel(self):
 
     def test_rule_expands_workspace_status_keys_in_wheel_metadata(self):
         filename = self._get_path(
-            "example_minimal_library_BUILD_USER_-0.1._BUILD_TIMESTAMP_-py3-none-any.whl"
+            "example_minimal_library{BUILD_USER}-0.1.{BUILD_TIMESTAMP}-py3-none-any.whl"
         )
 
         with zipfile.ZipFile(filename) as zf:
             self.assertAllEntriesHasReproducibleMetadata(zf)
             metadata_file = None
             for f in zf.namelist():
-                self.assertNotIn("_BUILD_TIMESTAMP_", f)
-                self.assertNotIn("_BUILD_USER_", f)
+                self.assertNotIn("{BUILD_TIMESTAMP}", f)
```

Comment:
I'm guessing the lowercase bug went undetected because this just looked for "not in" vs a more positive check about the value? Can we somehow check the values to verify the bug is gone?

Reply:
The filename in the line above is probably the assertion you seek for. The bug went undetected because we did not have a copy of this test with the new feature enabled.